recipe/meta.yaml

```diff
@@ -23,28 +22,33 @@ requirements:
     - libtool  # [unix]
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
-    - vs2017_{{ target_platform }}  # [win]
     - cmake
     - ninja
     - make  # [not win]
-    - python  # [win]
     - pkg-config  # [not win]
+    - llvm-openmp  # [osx]
   host:
-    - glpk
-    - libxml2
-    - suitesparse
-    - arpack   # [not win]
-    - gmp  # [not win]
-    - mpir  # [win]
-    - libblas
-    - liblapack
+    - glpk 4.65
+    - libxml2 {{ libxml2 }}
+    - suitesparse 7.0.1
```

Comment:
Conda-builds complains and says that it's not used:
```
   WARNING (igraph): run-exports library package pkgs/main::suitesparse-7.0.1-ha6b1e0b_0 in requirements/run but it is not used (i.e. it is overdepending or perhaps statically linked? If that is what you want then add it to `build/ignore_run_exports`)    
```
And looking at the code, I'm not even sure how it would work. They include `cs/cs.h` (see https://github.com/igraph/igraph/blob/0.10.4/src/core/sparsemat.c#L38), but our `suitesparse` package only exposes a `cs.h` file (withough a `cs` folder`...

What's more, the logs are saying
```
[1/344] Building C object vendor/cs/CMakeFiles/cxsparse_vendored.dir/cs_add.c.o
[2/344] Building C object vendor/cs/CMakeFiles/cxsparse_vendored.dir/cs_chol.c.o
[3/344] Building C object vendor/cs/CMakeFiles/cxsparse_vendored.dir/cs_amd.c.o
[4/344] Building C object vendor/cs/CMakeFiles/cxsparse_vendored.dir/cs_cholsol.c.o
[5/344] Building C object vendor/cs/CMakeFiles/cxsparse_vendored.dir/cs_compress.c.o
[6/344] Building C object vendor/cs/CMakeFiles/cxsparse_vendored.dir/cs_cumsum.c.o
[7/344] Building C object vendor/cs/CMakeFiles/cxsparse_vendored.dir/cs_counts.c.o
[8/344] Building C object vendor/cs/CMakeFiles/cxsparse_vendored.dir/cs_dfs.c.o
```
And on top of that, cmake says that `DIGRAPH_USE_INTERNAL_CXSPARSE` is not used.

Reply:
`suitesparse` is no longer a dependency for this package, so yes, it can be removed from the YAML file (along with its flag in the build script).

Reply:
Thanks